bin/docker-helper.sh

```diff
@@ -192,16 +202,11 @@ function cmd-push() {
       docker push $TARGET_IMAGE_NAME:$MAJOR_VERSION.$MINOR_VERSION.$PATCH_VERSION-$PLATFORM
     }
 
-    if [ -n "$FORCE_VERSION_TAG_PUSH" ] && [ "$FORCE_VERSION_TAG_PUSH" -eq "1" ]; then
-      echo "Force-enabled version tag push."
-      _push
-    elif [ -n "$FORCE_VERSION_TAG_PUSH" ] && [ "$FORCE_VERSION_TAG_PUSH" -eq "0" ]; then
-      echo "Force-disabled version tag push. Not pushing any other tags."
-    elif (git diff HEAD^ ${VERSION_FILE} | tail -n 1 | grep -v '.dev'); then
-      echo "Pushing version tags, version has changed in last commit."
+    if _is_release_commit; then
+      echo "Pushing version tags, we're building the commit of a version tag."
       _push
```

Comment:
Not part of this commit, but I think we could rename that function to be something more expressive, like `_push_versioned_tags` or something.